modules/network-observability-create-network-policy.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/configuring-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-network-policy_{context}"]
+= Secure Network Observability with a network policy using the web console
+You may need to create a network policy to secure ingress traffic to the 'netobserv' namespace. Using the web console, you can create a network policy 
+
```

Comment:
@stleerh In your video, you deleted a network policy and then added a new one. Will customers need to delete a pre-existing, default policy and add this new one?